patent.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+
+
+def patent(**kwargs):
+    patent_url = 'https://api.nasa.gov/patents/content'
+    get_params = {}
+    for key, value in kwargs.iteritems():
+        get_params[key] = value
+    get_response = requests.get(patent_url, params = kwargs)
+
+    if get_response.status_code == 429:
+        raise Exception('You have exceeded your rate limit')
+
+    body = get_response.json()
+
+    if 'error' in body:
```

Comment:
Is this how the Patents API reports errors? Genuinely curious, I haven't poked around it yet.